stixpy/timeseries/quicklook.py

```diff
@@ -517,32 +455,31 @@ def _parse_hdus(cls, hdulist):
             The path to the file you want to parse.
         """
         header = hdulist[0].header
-        control = Table(hdulist[1].data)
-        header["control"] = control
-        data = Table(hdulist[2].data)
-        energies = Table(hdulist[4].data)
-        dE = (
+        control = _hdu_to_qtable(hdulist[1])
+        data = _hdu_to_qtable(hdulist[2])
+        energies = _hdu_to_qtable(hdulist[4])
+        delta_energy = (
             energies[control["energy_bin_mask"][0]]["e_high"][-1] - energies[control["energy_bin_mask"][0]]["e_low"][0]
-            << u.keV
         )
         name = (
-            f'{energies[control["energy_bin_mask"][0]]["e_low"][0]}'
-            f'-{energies[control["energy_bin_mask"][0]]["e_high"][-1]}'
+            f'{energies[control["energy_bin_mask"][0]]["e_low"][0].value.astype(int)}'
+            f'-{energies[control["energy_bin_mask"][0]]["e_high"][-1].value.astype(int)} {energies["e_high"].unit}'
         )
 
         try:
-            data["time"] = Time(header["date_obs"]) + TimeDelta(data["time"] * u.s)
+            data["time"] = Time(header["date_obs"]) + TimeDelta(data["time"])
         except KeyError:
-            data["time"] = Time(header["date-obs"]) + TimeDelta(data["time"] * u.s)
+            data["time"] = Time(header["date-obs"]) + TimeDelta(data["time"])
 
-        data["variance"] = data["variance"].reshape(-1) / (dE * data["timedel"])
+        data["variance"] = data["variance"].reshape(-1) * u.ct / (delta_energy * data["timedel"].to(u.s))
```

Comment:
I'm actually not totally sure what units should be here TBH or if we should be correcting for live time.